File: Telerik/Reporting Q1 2014/Examples/CSharp/WinFormsDemo/MainForm.cs
namespace Telerik.Reporting.Examples.CSharp.WinFormsDemo
{
    using System.Windows.Forms;

    public partial class MainForm : Form
	{
        public MainForm()
		{
			InitializeComponent();
		}

        private void MainForm_Load(object sender, System.EventArgs e)
        {
            this.reportViewer1.RefreshReport();
        }
	}
}

File: Telerik/Reporting Q1 2014/Examples/VB/WinFormsDemo/MainForm.vb
Imports System.Windows.Forms
Imports Telerik.Reporting

Partial Public Class MainForm
    Inherits Form
    Public Sub New(ByVal report As IReportDocument)
        InitializeComponent()
    End Sub

    Private Sub MainForm_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        Me.reportViewer1.RefreshReport()
    End Sub
End Class
